### PR TITLE
[Core] Fixed payment history issue

### PR DESCRIPTION
**Pull request by NeverResponse**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Fixed tickets | #4110
| License       | MIT
| Doc PR        | -

### CODE REVIEW

File: src/Sylius/Bundle/CoreBundle/Controller/PaymentController.php [hunk 1]
```diff
@@ -41,15 +45,14 @@ public function historyAction(Request $request)
...
 
-        $view = $this
-            ->view()
-            ->setTemplate($this->config->getTemplate('history.html'))
-            ->setData([
-                $this->config->getResourceName() => $payment,
-                'logs' => $logRepository->getLogEntries($payment),
-            ])
-        ;
+        $view = View::create($configuration);
```
Comment (pjedrzejewski) [inline src/Sylius/Bundle/CoreBundle/Controller/PaymentController.php]: Why passing $configuration as argument?

Comment (pjedrzejewski) [inline src/Sylius/Bundle/CoreBundle/Controller/PaymentController.php]: First arg is data, which you override later.

Comment (NeverResponse) [inline src/Sylius/Bundle/CoreBundle/Controller/PaymentController.php]: :+1:

**Comment by pjedrzejewski** [comment]
Thank you Jasiek!

**Pull request merged**